docs/api-docs/management-apis/email-templates/email-template-examples.md

```diff
@@ -0,0 +1,116 @@
+# Email Templates Code Examples
+
+<div class="otp" id="no-index">
+
+### On this page
+
+- [Updating text](#updating-text)
+- [Adding a button](#adding-a-button)
+- [Changing logo size](#changing-logo-size)
+- [Changing font size](#changing-font-size)
+
+</div>
+
+This article provides examples of how to customize email templates. The examples are for use in the recently updated templates. You can enable the new version of transactional emails by going to **Marketing> Transactional Emails** and clicking **Try New Experience**. When you are ready to use a template, make sure you **Enable** the template from the respective dropdown. 
+
+Code examples below will demonstrate how to update text, add a button, change logo size, and change the font size.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+  
+  ### Note 
+  It is important to preview all changes and style your emails for accessibility.  
+  
+</div>
+</div>
+</div> 
+
+## Updating text
+To change existing text in an email template, you will need to update information in the **Phrases** and **Code** tabs. For detailed instructions, see [Editing Template Phrases](https://support.bigcommerce.com/s/article/Customizing-Emails?language=en_US#phrases). 
```

Comment:
We should try to avoid future tense and use present tense instead. 
```suggestion
To change existing text in an email template, update information in the **Phrases** and **Code** tabs. For detailed instructions, see [Editing Template Phrases](https://support.bigcommerce.com/s/article/Customizing-Emails?language=en_US#phrases). 
```